releasenotes/notes/fix-bug-for-setting-message-key-span-tag-from-kafka-serialization-classes-630b09004a97063a.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    kafka: This fix resolves an issue when setting the ``message.key`` span tag for kafka serializing producers and deserializing consumers. Serialized keys 
+    are now used for the tag value. If a key is already deserialized by the consumer, no ``message.key`` span tag is set as we do not have access to the
+    serialization function.
```

Can we simplify this a bit. We should also call out the error this fix resolves. 

```suggestion
    kafka: Resolves ``TypeError`` raised by serializing producers and deserializing consumers when the ``message.key`` tag is set on spans.
```